Storage/src/Lifecycle.php

```diff
@@ -90,7 +90,8 @@ public function __construct(array $lifecycle = [])
      *     @type string[] $matchesStorageClass Objects having any of the storage
      *           classes specified by this condition will be matched. Values
      *           include `"MULTI_REGIONAL"`, `"REGIONAL"`, `"NEARLINE"`,
-     *           `"COLDLINE"`, `"STANDARD"`, and `"DURABLE_REDUCED_AVAILABILITY"`.
+     *           `"ARCHIVE"`, `"COLDLINE"`, `"STANDARD"`, and
```

Comment:
I've got no issue reformatting to drop the double quotes (like on line 825 above), but if we do could we stay consistent and do that here as well please?